pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/actions/RestartDeclarativePipelineAction/index.jelly

```diff
@@ -31,7 +31,8 @@
             <j:if test="${it.restartEnabled}">
                 <h1>Restart ${it.owner.displayName} from Stage</h1>
                 <p>
-                    <j:out value="Help text to go here"/>
+                    <j:out value="${%A completed Declarative Pipeline run can be restarted, starting at the beginning of any top-level stage which ran in the original
+    build.}"/>
```

Comment:
Weird place to line-wrap? Anyway long text like this should go into an `index.properties`.

Reply:
Good point.